src/ffi/js/workspace.rs

```diff
@@ -42,21 +44,21 @@ impl Workspace {
 	}
 
 	/// List all available buffers in this workspace
-	#[napi(js_name = "filetree")]
-	pub fn js_filetree(&self, filter: Option<&str>, strict: bool) -> Vec<String> {
-		self.filetree(filter, strict)
+	#[napi(js_name = "search_buffers")]
```

Comment:
Change the js name of the function
```suggestion
	#[napi(js_name = "searchBuffers")]
```